File: euler6.java
public class SquareSum {
	public static void main(String[] args) {
		int sum_the_squares = 0, square_the_sum = 0;
		for (int i = 1; i <= 100 ; i++ ) {

			sum_the_squares+= i * i;
		}

		for (int i = 1; i <= 100 ; i++ ) {
			
			square_the_sum += i;
		}

		square_the_sum= square_the_sum * square_the_sum;
		System.out.println(square_the_sum - sum_the_squares);
	}
}

File: euler6.js
//Eh, why not. Solving the problem using JS. See corresponding HTML and completely unnessecary CSS files.

function SumTheSquares () {
	var summa = 0;
	for (var i = 1; i <= 100; i++) {
		summa += i*i;
	}
	
	return summa;
}

function SquareTheSum () {
	var summa = 0;
	var squaresum;
	for (var i = 1; i <= 100; i++) {
		summa += i;
	}
	squaresum = Math.pow(summa, 2);
	return squaresum;
}

document.write(SquareTheSum()-SumTheSquares());
